resources/lib/template.py

```diff
@@ -344,7 +344,10 @@ def getProperties( self, elem, items ):
                     continue
                     
                 # We've matched a property :)
-                properties[ name ] = property.text
+                if property.text is not None and property.text != "":
```

Comment:
Now, y'see, you put it like that and I understand ;) - I didn't get that I could them remove the `is not None` code - I really don't get Pythonic code :)